noir-projects/aztec-nr/aztec/src/state_vars/shared_mutable.nr

```diff
@@ -24,21 +23,11 @@ pub struct SharedMutable<T, let INITIAL_DELAY: u32, Context> {
     storage_slot: Field,
 }
 
-// Separators separating storage slot of different values within the same state variable
-global VALUE_CHANGE_SEPARATOR: u32 = 0;
-global DELAY_CHANGE_SEPARATOR: u32 = 1;
-global HASH_SEPARATOR: u32 = 2;
-
-// This will make the Aztec macros require that T implements the Serialize<N> trait, and allocate N storage slots to
-// this state variable. This is incorrect, since what we actually store is:
-//  - a ScheduledValueChange<T>, which requires 1 + 2 * M storage slots, where M is the serialization length of T
-//  - a ScheduledDelayChange, which requires another storage slot
-//
-// TODO https://github.com/AztecProtocol/aztec-packages/issues/5736: change the storage allocation scheme so that we
-// can actually use it here
+// This will make the Aztec macros require that T implements the Packable<N> and Eq traits, and allocate `N` storage
```

Comment:
```suggestion
// This will make the Aztec macros require that T implements the Packable and Eq traits, and allocate `N` storage
```

I'd just not mention the packing length to avoid confusing it with withhash's len